docs/sources/setup/datasource.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Configuring data source'
+menuTitle: Configure data source
+description: Configure the GitHub data source plugin to authenticate to GitHub
+keywords:
+  - data source
+  - github
+  - github repository
+  - API
+labels:
+  products:
+    - oss
+    - enterprise
+    - cloud
+weight: 103
+---
+
+# Configure data source
```

Comment:
The `title` and first heading should match for SEO and should include product names.

The `menuTitle` can omit the product name because it's implied by the website context.

```suggestion
title: Configure the GitHub data source plugin for Grafana
menuTitle: Configure
description: Configure the GitHub data source plugin to authenticate to GitHub
keywords:
  - data source
  - github
  - github repository
  - API
labels:
  products:
    - oss
    - enterprise
    - cloud
weight: 103
---

# Configure the GitHub data source plugin for Grafana
```